.github/workflows/check-licenses.yml

```diff
@@ -52,7 +52,6 @@ jobs:
       run: |
         echo "${MAVEN_CONFIG}" | tee ./.mvn/maven.config
     - name: Setup fossa-cli
-      # TODO: use main branch
       uses: camunda/infra-global-github-actions/fossa/setup@32470949e549396aa5f2c5def890de831b8ca107
```

Comment:
Just to double-check, you want to pin this version and not rely on `camunda/infra-global-github-actions/fossa/setup@main` like we do for most other actions?

Reply:
Yes. The rationale is to ensure (via Renovate PR) that the new version of the composite action works correctly and doesn't introduce any breaking changes, especially considering it's a new feature (we don't know how clean is FOSSA with their releases) potential impact on releases.